app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ListsController < ApplicationController
+
+    def new
+        @board = Board.find(params[:board_id])
+        @list = List.new
+    end
+
+    def create
+        @board = Board.find(params[:board_id])
+        @list = @board.lists.new
+        if @list.save
+            redirect_to board_path(@board), notice: 'List created'
+        else
+            render :new
+        end
+    end
+
+    def edit
+        @board = Board.find(params[:id])
+        @list = @board.lists.find_by(params[:id])
```

Comment:
To nie zadziala

Reply:
Uzyj metody do boarda

Reply:
Nadal nie dziala - nie powinno z find by

Reply:
nie powinnas miec metody która szuka i board i list 
to nie ma sensu.

musisz dobrze dobierac odpowiedzialnosci, zmiast tego zrób dwie osobne metody

Reply:
def new
        @board = Board.find(params[:board_id])
        @list = @board.lists.new
    end

u mnie jest tak

Reply:
czyli osobno set board i osobno set list?

Reply:
```
before_action :find_board, only: [:new]

def new
  @list = @board.lists.new
end
private 

def find_board
  @board = ...
end

def find_list
  @list = 
end
```

Reply:
NIE ZA DZIALAAAAAAAAA find_by przyjmuje klucz: wartosc a nie ID ! !!!!!

Reply:
?